plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -3181,6 +3185,16 @@ else if (object.getBukkitEntity() instanceof Hanging hanging) {
 
         if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_20)) {
 
+            // <--[tag]
+            // @attribute <EntityTag.body_armor>
+            // @returns ItemTag
+            // @description
+            // Returns the item equipped on an entity's body slot.
+            // -->
+            registerSpawnedOnlyTag(ItemTag.class, "body_armor", (attribute, entity) -> {
+                return new ItemTag(entity.getLivingEntity().getEquipment().getItem(EquipmentSlot.BODY));
+            }, "body_armour");

```

Comment:
A. the strings at the end of the register method are for deprecated variants of the tag, shouldn't have a deprecated variant on a new tag.
But also, is there any reason to have this equipment slot specifically as a separate tag? as opposed to just using the equipment property like the other equipment slots?